### PR TITLE
SM-1 first implementation

### PR DESCRIPTION
**Pull request by MannaNebesnaya**

### CODE REVIEW

File: src/main/java/com/byndyusoft/java/spring/microservice/template/utils/package-info.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * A place for non-business logic.
+ * <li>helpers for java types and structures: String, Integer, Map etc
+ * <li>helper for frameworks and standards: json, xml, wrappers
+ */
+
+package com.byndyusoft.java.spring.microservice.template.utils;
```
Comment (wisedog706) [inline src/main/java/com/byndyusoft/java/spring/microservice/template/utils/package-info.java:7]: remove utils, its package for garbage

File: src/main/java/com/byndyusoft/java/spring/microservice/template/config/package-info.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Place for bean configuration {@link org.springframework.context.annotation.Configuration}.
+ *
+ */
+
+package com.byndyusoft.java.spring.microservice.template.config;
```
Comment (wisedog706) [inline src/main/java/com/byndyusoft/java/spring/microservice/template/config/package-info.java]: config package already have class from this package, why this stub need? (also `exception`, `model`, `repository`, `service`, `controller`, `handler` packages)

File: src/main/java/com/byndyusoft/java/spring/microservice/template/client/package-info.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/**
+ * A place for external integrations in which the service acts as a client.
+ */
+
+package com.byndyusoft.java.spring.microservice.template.client;
```
Comment (wisedog706) [inline src/main/java/com/byndyusoft/java/spring/microservice/template/client/package-info.java:5]: if we add package for cients, maybe add template classes too?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
...
+* Java 17
+* Spring Boot 2.7.5
+* Spring Cloud 2021.0.5
+* Maven 3.6.3
+
+## OpenAPI Specification
+Please add OpenAPI specification if exists. Currently automatic generation.
+
+## Database schema
+Generated database schema can be found [here](./db-schema.svg).
```
Comment (wisedog706) [inline README.md:15]: why db in template?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,54 @@
-# java-spring-microservice-template
-Seed repo for all new services 
+# Microservice template
+## Summary
+Seed repo for all new services
```
Comment (wisedog706) [inline README.md:3]: `Template repo` better imo

**Review by wisedog706 — CHANGES_REQUESTED** [review]
And what is `SM` meaning?

File: README.md (hunk 4 above)
Comment (MannaNebesnaya) [reply README.md:15]: done

File: README.md (hunk 5 above)
Comment (MannaNebesnaya) [reply README.md:3]: Discussed

File: README.md (hunk 4 above)
Comment (MannaNebesnaya) [reply README.md:15]: Discussed

File: src/main/java/com/byndyusoft/java/spring/microservice/template/client/package-info.java (hunk 3 above)
Comment (MannaNebesnaya) [reply src/main/java/com/byndyusoft/java/spring/microservice/template/client/package-info.java:5]: Discussed

File: src/main/java/com/byndyusoft/java/spring/microservice/template/config/package-info.java (hunk 2 above)
Comment (MannaNebesnaya) [reply src/main/java/com/byndyusoft/java/spring/microservice/template/config/package-info.java]: Discussed

File: src/main/java/com/byndyusoft/java/spring/microservice/template/utils/package-info.java (hunk 1 above)
Comment (MannaNebesnaya) [reply src/main/java/com/byndyusoft/java/spring/microservice/template/utils/package-info.java:7]: Discussed

**Pull request merged**